RAMP/ramp.py

```diff
@@ -72,10 +72,11 @@ def unpack(bundle):
 @click.option('--redis-pack-min-version', '-R', 'redis_pack_min_version', default=module_metadata.MIN_REDIS_PACK_VERSION, help='redis pack minimum version')
 @click.option('--config-command', '-cc', default=module_metadata.CONFIG_COMMAND, help='command used to configure module args at runtime')
 @click.option('--os', '-O', default=module_metadata.OS, help='build target OS (Darwin/Linux)')
+@click.option('--os_list', '-ol', default=module_metadata.OS_LIST, help='list of supported os')
```

Comment:
- Why maintain two fields?
- What about CPU arch? (x86/ 386/ ppc...)

Reply:
I didn't want to remove `os` since I don't know if the cluster relies on it

https://github.com/RedisLabs/RAMP/blob/master/RAMP/module_metadata.py#L5

Reply:
@ushachar @swilly22 @itamarhaber can we remove the os flag?